### PR TITLE
MES 699 cpc end test summary

### PR DESCRIPTION
**Pull request by Cullima**
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2020-06-09 at 15 52 19

### CODE REVIEW

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export class ModalResultItemComponent {
+
+  @Input()
+  label: string;
+
+  @Input()
+  score: number;
+
+  @Input()
+  isPass: number;
```
Comment (matthew2564) [inline src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts]: checking if 'isPass' is a number?

Comment (Cullima) [reply src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts]: no, thanks it should be boolean

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { IonicPageModule } from 'ionic-angular';
+import { CpcEndTestModal } from './cpc-end-test-modal';
+import { ModalResultItemComponent } from './components/modal-result-item/modal-result-item';
+
+@NgModule({
+  declarations: [
+    CpcEndTestModal,
```
Comment (matthew2564) [inline src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts]: should we look at upper casing the category in same format as pages

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  ],
+  imports: [
+    IonicPageModule.forChild(CpcEndTestModal),
+  ],
+  exports: [
+    CpcEndTestModal,
+    ModalResultItemComponent,
+  ],
+})
+export class CpcEndTestModalModule { }
```
Comment (matthew2564) [inline src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts]: same as above in regards to casing

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  getOutcomeIcon(): string {
+    const passImage = '../../../../../../../assets/imgs/greenCorrectAnswer.png';
+    const failImage = '../../../../../../../assets/imgs/redWrongAnswer.png';
```
Comment (matthew2564) [inline src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts]: in other places where an icon is displayed the HTML references just 'assets/imgs/greenCorrectAnswer.png' for example, does this still work if the path is:

 const passImage = 'assets/imgs/greenCorrectAnswer.png';

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts (hunk 1 above)
Comment (Cullima) [reply src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts]: changed

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts (hunk 3 above)
Comment (Cullima) [reply src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts]: changed

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts (hunk 4 above)
Comment (Cullima) [reply src/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts]: changed

File: src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts (hunk 2 above)
Comment (Cullima) [reply src/pages/test-report/cat-cpc/components/cpc-end-test-modal/cpc-end-test-modal.module.ts]: changed

**Review by andrewsetterfield — APPROVED** [review]

**Review by matthew2564 — APPROVED** [review]

**Pull request merged**